### PR TITLE
[YUNIKORN-1154] Translate YuniKorn Release Procedure to zh-cn

### PR DESCRIPTION
**Pull request by cdmikechen**
There are two task to do in this PR
- [x] Translate YuniKorn Release Procedure to zh-cn
- [x] Fix some problems in YuniKorn Release Procedure 

image

### CODE REVIEW

File: src/pages/community/release_procedure.md [hunk 1]
```diff
@@ -223,7 +225,7 @@ Older releases should be downloaded from the archive directly, not from the rele
 
 The releases need to clean up in two locations:
 * Remove the newly released version from the _dev_ area by removing the old release candidate directory.
-  For the location see [release candidate location](#Upload-Release-Candidate-Artefacts)
+  For the location see [release candidate location](#Upload-Release-Candidate-Artifacts)
 * Remove the non-current release from the _release_ area by removing the old release directory.
   For the location see [release location](#Publish-the-Release)
```
Comment (wilfred-s) [inline src/pages/community/release_procedure.md:230]: this one needs fixing also: only lower case

**Review by wilfred-s — CHANGES_REQUESTED** [review]
now that you are touching the english version you need to fix all broken links: only lower case references are supported in the links

File: i18n/zh-cn/docusaurus-plugin-content-pages/community/release_procedure.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# YuniKorn 发布程序
+
+[发布存储库](https://github.com/apache/yunikorn-release) 包含了 Apache YuniKorn 创建 release（下文统称为发布）的代码和配置。
```
Comment (yangwwei) [inline i18n/zh-cn/docusaurus-plugin-content-pages/community/release_procedure.md:25]: >Apache YuniKorn 创建 release（下文统称为发布）

Release 是不是可以翻译为 ”发行版” ？

File: i18n/zh-cn/docusaurus-theme-classic/navbar.json [hunk 3]
```diff
@@ -28,6 +28,9 @@
     },    
     "item.label.Reporting Issues": {
         "message": "报告问题"
+    },     
+    "item.label.Release Procedure": {
+        "message": "发布程序"
```
Comment (yangwwei) [inline i18n/zh-cn/docusaurus-theme-classic/navbar.json:33]: 创建发行版流程

**Review by yangwwei — CHANGES_REQUESTED** [review]

**Comment by wilfred-s** [comment]
I just noticed that we have lost an important update in the English version. YUNIKORN-994 fixed the tagging documentation. That change was not picked up when moving the doc from the release repo to the web site: YUNIKORN-1166 has been filed to fix that. I will create a PR later today.

**Comment by wilfred-s** [comment]
Sorry for the major update to the doc but there were a number of issues that needed to be resolved.
Changes are committed to master.

**Comment by cdmikechen** [comment]
@wilfred-s @yangwwei 
OK~ I'll revise it again in these two days.

**Comment by cdmikechen** [comment]
@yangwwei
目前可能有2个单词我自己会觉得有一些争议点，一个是 `release`，一个是 `artifact`。我在microsoft的azure文档中找到了一个相关文档的翻译。在文档中，他们将 `Release artifacts` 翻译为 `发布工件`
https://docs.microsoft.com/zh-cn/azure/devops/pipelines/release/artifacts?view=azure-devops
https://docs.microsoft.com/en-us/azure/devops/pipelines/release/artifacts?view=azure-devops

我个人觉得从阅读感觉上来说，`release` 可以暂时沿用 `发布` 这一个动词概念，部分名词也可以复用 `发布`，然而还有一些地方翻译起来可能会有些混乱的情况下，我暂时还是保持 `release` 的原语义。很多阅读者应该即使不翻译 release 也能理解在文中的含义。
`artifact` 的解释我觉得可能阅读感觉上会差一些，即使翻译成 `构件` 或者其他一些相似含义也觉得很别扭。我后来考虑如果把 `Release artifacts` 翻译为 `发布文件` 似乎阅读理解的感觉上会好一点。

**Review by cheersyang — APPROVED** [review]
+1

**Review by yuchaoran2011 — APPROVED** [review]

**Pull request merged**